packages/rest/src/parser.ts

```diff
@@ -124,5 +126,14 @@ function buildOperationArguments(
     }
   }
   if (requestBodyIndex > -1) paramArgs.splice(requestBodyIndex, 0, body);
-  return paramArgs;
+
+  debug('Coercing parameters', paramArgs);
+
+  const coercedParamArgs: OperationArgs = [];
+  for (const arg of paramArgs) {
+    debug('Coercing parameter', arg);
+    coercedParamArgs.push(typeof arg === 'object' ? arg : coerce(arg));
```

Comment:
Strictly speaking, it's about deserializing parameters from an http request. We have to respect the parameter type spec as illustrated in https://github.com/strongloop/loopback-next/pull/941.

Reply:
That makes sense, I was working on using the deserializers from `@repository` and was really wondering why we couldn't use a simple coercer. I'll revert this to what it is in the original PoC